.travis.yml

```diff
@@ -25,9 +31,20 @@ matrix:
 notifications:
   email: false
 script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("Memento")'
   - ./.travis/test.sh
 after_success:
-  - julia -e 'cd(Pkg.dir("Memento")); Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())'
-  - julia -e 'cd(Pkg.dir("Memento")); Pkg.add("Documenter"); include(joinpath("docs", "make.jl"))'
+  - |
+      julia -e '
+        VERSION >= v"0.7.0-DEV.3656" && using Pkg
+        VERSION >= v"0.7.0-DEV.5183" || cd(Pkg.dir("Memento"))
+        Pkg.add("Coverage")
+        using Coverage
+        Codecov.submit(Codecov.process_folder())
+      '
+  - |
+      julia -e '
+        VERSION >= v"0.7.0-DEV.3656" && using Pkg
+        VERSION >= v"0.7.0-DEV.5183" || cd(Pkg.dir("Memento"))
+        Pkg.add("Documenter")
+        include(joinpath("docs", "make.jl"))
+      '
```

Comment:
I sometimes combine these lines so I don't require the double VERSION checks

Reply:
I personally prefer having coverage submission and documentation deployment as separate steps, even if it means duplicating some logic until we drop 0.6.